sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -1989,7 +1989,7 @@ pi_result piPluginInit(pi_plugin *PluginInit) {
   }
 
   // Check that the major version matches in PiVersion and SupportedVersion
-  static int PiMajorVersionLen = strcspn(PluginInit->PiVersion.".");
+  static int PiMajorVersionLen = strcspn(PluginInit->PiVersion, ".");
```

Comment:
Other plugins need this too, and so maybe again move this to pi.h

Reply:
Change made. Using macro `_PI_VERSION_MAJOR_STRING_LEN` to define the length of Major part of string